locales/en/translation.json

```diff
@@ -80,6 +80,7 @@
         "width": "Don't use the `width` attribute. Instead, use the CSS `width` property"
       },
       "lower-case-attribute-name": "Use lower case for attribute names",
+      "invalid-list-children": "The `<{{-tag}}>` tag must contain `<{{-children}}>`",
```

Comment:
@outoftime I think this error message is a bit confusing if you have written some valid `<li>` tags, but then wrote some text outside of any tag. I saw a student do this last week for example. Suggestions on wording? Maybe something like

`All text inside the <{{-tag}}> must be wrapped by <{{-children}}>`

Reply:
Definitely agree with your line of thinking, not sure how recognizable the word “wrapped” will be in this context though. Maybe “surrounded” instead?

Reply:
I've made another update to the wording, in the latest it will be `The text asdf inside the <ul> tag must be surrounded by <li> tags`